### ISSUE TITLE
MSI installer upgrade UI is confusing

### ISSUE THREAD

**Issue by Avispa** [mumble-voip/mumble#2901]
Referring to a side, or the main issue probably, of #2900.
The (msi) installer for Mumble 1.3.0.1937 (g289d0d4) uninstalls old Mumble version without any warning or any mean.

#### Reproduce:
* Install Mumble 1.2 (for example) to directory **A**
* Install Mumble 1.3 snapshot to directory **B**

#### Result:
Directory *A* is found empty except for the certificate file. Conclusion: uninstall routine got executed silently.
Side effect by the view of this issue (as noticed in #2900), as a large assumption, the installation in **B** is confused because the library search path is wrong. Probably pointing to directory **A**(?). Again just a large assumption.

#### Expectation:
The obvious:
* Don't delete present installs when not asked for.
* Having both installs running side by side (when different installation directory is selected).

**Comment by mkrautz** [comment]
This is also the case for 1.2.18 -> 1.2.19, fwiw. So it's not a regression. But still a confusing issue.

**Comment by Avispa** [comment]
When I choose a custom install directory it cannot resp. should not delete something in another place. More over not silently! This is not only confusing this is just mall functioning.
I understand the intention and it is somehow arguable when installed in the same place but again: if I install **foo** in one place it is wrong to delete **$bar** in another place.

**Comment by Kissaki** [comment]
This is not foo and bar, this is an old and new version of the same program.

Obviously there's a use case for installing versions in parallel, but pretty much not at all for the normal end user. Also obviously, this doesn't make as much sense for development snapshots. But it's not nearly as clearly wrong as you say it is.

**Comment by mkrautz** [comment]
The problem here is actually that the installer doesn't present the upgrade as an upgrade.
It seems like a fresh install.
So I get why @Avispa would expect *both* directories to stay.

**Comment by mkrautz** [comment]
OK, this was not fixed by 346702fc37296a9ddf482b484cf15e58216b7c4a.

I suppose the problem here is that the MSI UI is not good at showing that it's an upgrade that is taking place. For example, why would it allow you to change the path if it's just upgrading? That doesn't make much sense to me.

**Comment by Krzmbrzl** [comment]
The installer has been completely reworked. If there are still issues with it, please create a new issue describing what you are missing.

Closing as outdated.
